schematic/store/synapse.py

```diff
@@ -476,3 +476,63 @@ def associateMetadataWithFiles(self, metadataManifestPath: str, datasetId: str)
 
         return manifestSynapseFileId
 
+
+    def getFileAnnotations(self, fileId: str) -> dict:
```

Comment:
Trivial note: We can use `Dict[str, str]` from the `typing` module instead of `dict` here?

Reply:
Good catch!